lib/croutons/breadcrumb_trail.rb

```diff
@@ -15,16 +15,16 @@ def initialize(template_identifer, objects = {})
       build_breadcrumbs
     end
 
-    def method_missing(name, *args)
+    def method_missing(name, *args, &block)
```

Comment:
I would say that its purpose is first for clarity, and second for the call to `super` (which is irrelevant because the block is passed anyway, so mostly clarity)